.github/workflows/build.yml

```diff
@@ -220,17 +220,7 @@ jobs:
           gpg-passphrase: GPG_PASSPHRASE
       - name: Run
         run: |
-          ./gradlew :src:compiler:core:publish \
-                    :src:compiler:lib:publish \
-                    :src:converter:avro:publish \
-                    :src:converter:openapi:publish \
-                    :src:plugin:gradle:publish \
-                    :src:plugin:maven:publish \
-                    :src:plugin:arguments:publish \
-                    :src:integration:jackson:publish \
-                    :src:integration:wirespec:publish \
-                    :src:integration:spring:publish \
-                    :src:tools:generator:publish
+          ./gradlew publish
```

Comment:
Please note that the following artifacts are missing here:

- `src:integration:avro:publish`
- `:src:plugin:cli:publish`
- `:src:ide:intellij-plugin:publish`
- `:src:plugin:npm:publish`

I know that we are definitely missing the first one (`src:integration:avro:publish`) as this is expected, and also used in an example. 

For the others, I've assumed for now that we explicitly don't want them published on `maven-central`,  as you (will) see in the rest of the code changes. They are explicitly excluded as follows:

```kts
plugins.withId("maven-publish") {
    tasks.configureEach {
        enabled = false
    }
}
```


Is this also what we want @wilmveel , @jerrevanveluw ?